docs/User-Data-Store-API.md

```diff
@@ -0,0 +1,945 @@
+|----------------------------------------------------------------|-----------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
+| `userId`<span class="required" title="Required">*</span>       | `String`                                | User identifier of document owner.                                                                                              |
+| `documentType`<span class="required" title="Required">*</span> | `String`                                | One of `profile`, `personal_id`, `passport`, `drivers_license`, `payment_card`, `loyalty`, `photo`.                             |
+| `dataType`<span class="required" title="Required">*</span>     | `String`                                | One of `claims`, `jwt`, `vc`, `image_base64`, `binary_base64`, `url`.                                                           |
+| `documentDataId`                                               | `String`                                | Optional identifier of the data stored in the document (e.g. ID card number).                                                   |
+| `externalId`                                                   | `String`                                | Optional external identifier of the document (e.g. identifier used in the bank system).                                         |
+| `documentData`<span class="required" title="Required">*</span> | `String`                                | Document data. Use `{}` for empty data.                                                                                         |
+| `attributes`<span class="required" title="Required">*</span>   | `Map<String, Object>`                   | Attributes containing metadata for the document. Use `{}` for empty attributes.                                                 |
+
+#### Response 200
+
+```json
+{
+  "status": "OK"
+}
+```
+<!-- end -->
+
+<!-- begin api DELETE /admin/documents -->
+### Delete Documents
+
+Delete documents.
+
+<!-- begin remove -->
+
+<table>
+    <tr>
+        <td>Method</td>
+        <td><code>DELETE</code></td>
+    </tr>
+    <tr>
+        <td>Resource URI</td>
+        <td><code>/admin/documents</code></td>
+    </tr>
+</table>
+<!-- end -->
+
+#### Request
+
+- Headers:
+    - `Authorization: Basic ...`
+
+##### Query Params
+
+| Param        | Type                                                     | Description                                                         |
+|--------------|----------------------------------------------------------|---------------------------------------------------------------------|
+| `userId`     | `String`<span class="required" title="Required">*</span> | User identifier of the owner of deleted documents.                  |
```

Comment:
As it is in the request, the documents are yet to be deleted. This past tense is used in more places.

```suggestion
| `userId`     | `String`<span class="required" title="Required">*</span> | User identifier of the owner of documents to be deleted.                  |
```